force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -71,7 +78,17 @@ public with sharing class FlowControlLoopsCollections {
      * @return A string indicating whether the number is "Positive", "Negative", or "Zero", or null if the number is null.
      */
     public static String checkNumber(Integer a) {
-        return null; // Replace null with the variable you used to store the result
+        if (a > 0) {
+            return 'Positive';
+        }
+        else if (a < 0) {
+            return 'Negative';
+        }
+        else if (a == 0) {
+            return 'Zero';
+        } else {
+            return null; // Replace null with the variable you used to store the result
+        }
```

Comment:
I would do the null check at the beginning: what happens if a == null?
if(a == null) {
return null;
} then continue your work so that you can evaluate a > and a <